examples/with-next-auth/README.md

```diff
@@ -0,0 +1,50 @@
+# next-auth
+
+Example with [`next-auth`](https://github.com/iaincollins/next-auth) integration.
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com/now):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/vercel/next.js/tree/canary/examples/with-next-auth)
+
+## How to use
+
+A Prerequisite here is that you provide some sort of authentication mechanism, be that in the form of OAuth keys/secrets from a provider (Google, Twitter, etc.) or an SMTP connection string to enable email authentication.
+
+You must copy the `example.env` file and create your own `.env` file and fill out at least one of the authentication provider variables.
```

Comment:
rename `example.env` to `.env.local.example` and use `.env.local` instead of `.env`

Reply:
@iaincollins what do you think?

Reply:
Sure! Using`.env` as a file extension makes the most sense to me personally (e.g. `local.env`) but I'm happy to go with whatever conventions the Next.js team like in their repo/examples so the examples in this repo are consistent with each other!

I assume the new `env var` handling in Next.js picks up the suggested file name automatically without additional code/configuration?

Reply:
`.env` is intended to be public (i.e can be pushed to the repo), so it's not a good place for secrets. If you have environment variables that don't have any secret value feel free to add them in `.env`.

Reply:
Okay so I've renamed the .env.local.example file and adjusted the instructions in the latest commit

Reply:
I still believe the process of adding this configs is not clear enough in the readme 🤔